src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -69,8 +65,10 @@ const useGroupSetting = ({route}) => {
       };
       if (urlImage) {
         try {
-          const res = await uploadFile(`data:image/jpeg;base64,${base64Profile}`);
-          body = {...body, channel_image: res.data.url};
+          const res = await uploadPhotoImage(urlImage);
+          if (res) {
+            body = {...body, channel_image: res.data.url};
+          }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code for uploading an image has been updated to use the `uploadPhotoImage` function from the `useUploadImage` hook. This change seems appropriate as it aligns with the PR title indicating that a method for uploading compressed images has been added.
```diff
- const res = await uploadFile(`data:image/jpeg;base64,${base64Profile}`);
+ const res = await uploadPhotoImage(urlImage);
  if (res) {
    body = {...body, channel_image: res.data.url};
  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->